FBC/FunctionalEntities/NorthAmericanEntities/USRegulatoryAgencies.rdf

```diff
@@ -1857,6 +1868,24 @@ The ABA RTN is necessary for the Federal Reserve Banks to process Fedwire funds
 		<lcc-lr:identifies rdf:resource="&fibo-fbc-fct-usjrga;SouthDakotaBusinessInformationRegistry"/>
 	</owl:NamedIndividual>
 	
+	<owl:Class rdf:about="&fibo-fbc-fct-usjrga;TaxpayerIdentificationNumber">
+		<rdfs:subClassOf rdf:resource="&fibo-be-le-fbo;TaxIdentifier"/>
+		<rdfs:label>taxpayer identification number</rdfs:label>
+		<skos:definition>identification number used by the Internal Revenue Service (IRS) in the administration of tax laws in the United States</skos:definition>
+		<fibo-fnd-utl-av:abbreviation>TIN</fibo-fnd-utl-av:abbreviation>
+		<fibo-fnd-utl-av:adaptedFrom rdf:datatype="&xsd;anyURI">https://www.irs.gov/individuals/international-taxpayers/taxpayer-identification-numbers-tin</fibo-fnd-utl-av:adaptedFrom>
+		<fibo-fnd-utl-av:explanatoryNote>It is issued either by the Social Security Administration (SSA) or by the IRS. A Social Security number (SSN) is issued by the SSA whereas all other TINs are issued by the IRS.
```

Comment:
Do we even have SSN (could not find it)?

Reply:
Generally comments should be on the subclass not the superclass.

Reply:
No - there are two more we need - SSN and an adoption taxpayer identification number, but I would add those here under a different issue.  The BE issues were more concerned about business entities.